modules/deploy/partials/kubernetes/guides/deploy-redpanda.adoc

```diff
@@ -139,47 +127,37 @@ helm repo add jetstack https://charts.jetstack.io
 helm repo update
 helm install cert-manager jetstack/cert-manager \
   --set installCRDs=true \
-  --namespace cert-manager  \
+  --namespace cert-manager \
   --create-namespace
 ```
 +
 TLS is enabled by default. The Redpanda Helm chart uses cert-manager to manage TLS certificates by default.
 
 . Install Redpanda with SASL enabled:
 +
-```bash
+[,bash,lines=6+8+9]
+----
 helm repo add redpanda https://charts.redpanda.com
 export DOMAIN=customredpandadomain.local && \
-helm install redpanda redpanda/redpanda --namespace <namespace> --create-namespace \
+helm install redpanda redpanda/redpanda \
+  --namespace <namespace> --create-namespace \
   --set auth.sasl.enabled=true \
   --set "auth.sasl.users[0].name=superuser" \
   --set "auth.sasl.users[0].password=secretpassword" \
-  --set external.domain=${DOMAIN} --wait \
+  --set external.domain=${DOMAIN} \
+  --set "storage.persistentVolume.storageClass=csi-driver-lvm-striped-xfs" \
+  --wait \
   --timeout 1h
-```
-+
-Here, you create a superuser called `superuser` that can grant permissions to new users in your cluster using access control lists (ACLs). You also define a custom domain that each broker will advertise externally.
+----
 +
+- `external.domain`: The custom domain that each broker will advertise to clients externally. This domain is added to the internal and external TLS certificates so that you can connect to the cluster using this domain.
```

Comment:
```suggestion
- `external.domain`: The custom domain that each broker advertises to clients externally. This domain is added to the internal and external TLS certificates so that you can connect to the cluster using this domain.
```